src/cljs-lib/src/calva/inspector.cljs

```diff
@@ -0,0 +1,43 @@
+(ns  calva.inspector
+  "Utilities for supporting Calva's inspection of data structures."
+  (:require [calva.js-utils :refer [jsify parse-edn]]))
+
+(defn- annotate
```

Comment:
I'd make this fn public and write a unit-test or two for it, because it's pure and all.